addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -62,17 +85,19 @@ def onchange_project_id(self):
             # reset task when changing project
             self.task_id = False
 
-    @api.onchange('task_id')
-    def _onchange_task_id(self):
-        if not self.project_id:
-            self.project_id = self.task_id.project_id
+    @api.depends('task_id.project_id')
+    def _compute_project_id(self):
+        for record in self:
```

Comment:
```python
for timesheet in self: ...
```
Better to know what we are dealing with ;)
Everywhere.

Reply:
This was already done. I think you have reviewed an old version of this file.